src/OrleansLoggerProvider/LegacyOrleansLogger.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+
+namespace Orleans.Extensions.Logging.Legacy
+{
+    /// <summary>
+    /// LegacyOrleansLogger supports legacy orleans logging features, including <see cref="ILogConsumer"/>, <see cref="ICloseableLogConsumer">,
+    /// <see cref="IFlushableLogConsumer">, <see cref="Severity">. 
+    /// </summary>
+    [Obsolete("Use Microsoft.Extensions.Logging built-in loggers")]
```

Comment:
I think we should rephrase it, because we encourage people to use ```ILogger``` calls to log everything but not to use builtin loggers, since they can use ANY conforming logger.

Reply:
What do you think the words need to be? please use ILogger? it is kind of confusing, since LegacyOrleansLogger is an Ilogger. 

please tell me the words you think fit here

Reply:
How about "The Microsoft.Orleans.Logging.Legacy namespace was kept to facilitate migration from Orleans 1.x but will be removed in the near future. It is recommended that you use the Microsoft.Extensions.Logging infrastructure and providers directly instead of Microsoft.Orleans.Logging.Legacy.Logger and Microsoft.Orleans.Logging.Legacy.ILogConsumer."

(I understand you haven't migrated Logger yet, but you can skip that part for now).
Again, I'd put this in a constant, and refer to it from every Obsolete attribute in this namespace.

Reply:
sounds good. except for it is kind of long lol

Reply:
Heh, but it's very explicit. BTW, you keep duplicating the string everywhere. Why are you not creating the constant and have the string defined just once?

Reply:
hmmm..... insert texts explaining I know what i'm doing but in fact not.

Reply:
fixed